production/mimir/mimir.libsonnet

```diff
@@ -0,0 +1,46 @@
+(import 'ksonnet-util/kausal.libsonnet') +
```

Comment:
I'm guessing the upstream jsonnet is too complex/overkill for this?

Reply:
Are you referring to pulling in [the jsonnet lib that is distributed with Mimir](https://github.com/grafana/mimir/tree/main/operations/mimir)? If so, I'd say it's overkill for this. 

Also, Tempo and Loki don't use the corresponding upstream jsonnet here either, so I'm favouring local consistency within the repo over reuse.

Reply:
Yep, exactly. Was more a curiosity thing here. Makes sense :-)